frontend/explorer/public/js/main.js

```diff
@@ -119,6 +120,139 @@ function appendUnlabeledStat(table, text) {
 }
 
+
+// appendNavigationButtons appends two buttons that navigate between the previous and the next block.
+/* function appendNavigationButtons(element, explorerBlock) {
+	var buttonContainer = document.createElement('div');
+	buttonContainer.classList.add('navigation-buttons');
+
+	var previousButton = document.createElement('button');
+	previousButton.id = 'previousbutton';
+	previousButton.textContent = 'Previous Block';
+	if (explorerBlock.height == 0) {
+		previousButton.disabled = true;
+	}
+
+	var nextButton = document.createElement('button');
+	nextButton.id = 'nextbutton';
+	nextButton.textContent = 'Next Block';
+
+	buttonContainer.appendChild(nextButton);
+	buttonContainer.appendChild(previousButton);
+	element.appendChild(buttonContainer);
+
+	previousButton.onclick = (e) => {
+		goToPreviousBlock(explorerBlock.height);
+	}
+	
+	nextButton.onclick = (e) => {
+		goToNextBlock(explorerBlock.height);
+	}
+
+} */
+
+function appendNavigationElements(element, explorerBlock) {
+	//create all needed elements
+	var buttonContainer = document.createElement('div');
+	var searchButtonForm = document.createElement('form');
+	var previousButton = document.createElement('button');
+	var nextButton = document.createElement('button');
+	var searchButton = document.createElement('button');
+	var searchField = document.createElement('INPUT'); 
+
+	//add ID's to elements
+	buttonContainer.id = 'navigation-buttons-container';
+	searchButtonForm.id = 'search-button-form';
+	previousButton.id = 'button-previous';
+	nextButton.id = 'button-next';
+	searchButton.id = 'search-button';
+	searchField.id = 'search-field-input';
+
+	//add text to the buttons
+	previousButton.textContent = 'Previous Button';
+	nextButton.textContent = 'Next Button';
+	searchButton.textContent= 'Go To Block';
+
+	//set attributes to searchField
+	searchField.required = true;
+	searchField.setAttribute('type', 'text');
+	searchField.setAttribute('name', 'height');
+	searchField.setAttribute('placeholder', explorerBlock.height);
+
+	//set attributes to searchButton
+	searchButton.setAttribute('value', 'go');
+	searchButton.setAttribute('type', 'submit');
+
+	//set attributes to searchButtonForm
+	searchButtonForm.setAttribute('method', 'get');
+	searchButtonForm.setAttribute('action', 'block.html');
+
+	//add elements to searchButtonForm
+	searchButtonForm.appendChild(searchButton);
+	searchButtonForm.appendChild(searchField);
+
+	//add elements to buttonContainer
+	buttonContainer.appendChild(previousButton);
+	buttonContainer.appendChild(nextButton);
+	buttonContainer.appendChild(searchButtonForm);
+
+	//add container to element
+	element.appendChild(buttonContainer);
+
+	//add click-event to previousButton
+	previousButton.onclick = (e) => {
+		goToPreviousBlock(explorerBlock.height);
+	}
+
+	//add click-event to nextButton
+	nextButton.onclick = (e) => {
+		goToNextBlock(explorerBlock.height);
+	}
+}
+
+function goToNextBlock(height) {
+	var nextBlockHeight = ++height;
```

Comment:
please first check if that block exists (by trying to fetch it based on its height, as to not send the user to a useless page, If the block doesn't exist, just reload this page or so (see: https://github.com/threefoldfoundation/tfchain/pull/175#issuecomment-414940345)

Reply:
Disabled the next-button when user is on the page of the latest block.